evap/staff/fixtures/excel_files_test_data.py

```diff
@@ -208,7 +208,7 @@
     ['Prof. Dr.', 'Prorsus', 'Christoph', '123@external.com']
 ]
 
-random_file_content = b"Hallo Welt\n"
```

Comment:
Just found out, why this was necessary: `fixtures` wasn't a python module before, so pylint ignored it